src/electionguard-ui/ElectionGuard.UI.Lib/Services/DbService.cs

```diff
@@ -42,6 +42,7 @@ public static class DbService
 
     private static string DbHost = string.Empty;
     private static string DbPassword = string.Empty;
+    private static string DbConnection = string.Empty;

```

Comment:
These should be moved into `Init(host, password)` as constants / parameters.

Reply:
per convo, this will be left as private statics for future reconnect logic

Reply:
reconnect method added